docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -846,7 +827,18 @@ these pieces of data from it. It then performs the following actions:
   
 </Admonition>
 
-## Step 5/5. Remove trust between your clusters
+## Step 5/6. Updating role mappings
+
+You can update the role mappings by modifying the trusted_cluster.yaml file we created
+earlier to contain the new role mapping.
+
+After logging into the leaf cluster:
+
+```code
+$ tctl create --force updated_cluster.yaml
```

Comment:
```suggestion
$ tctl create --force trusted_cluster.yaml
```

Either keep the filename the same, or tell the user that you expect them to save updates to a different file. Just changing the name might confuse the reader.